microservices/s3_to_redshift/build_derived_assets.py

```diff
@@ -175,8 +208,10 @@
         except psycopg2.Error:
             logger.exception((
                 'Error: failed to execute the transaction '
-                'to prepare the gov_assets_derived PDT'))
+                'to prepare the {schema_name}.asset_downloads_derived PDT')
+                .format(schema_name=schema_name))
```

Comment:
can you make this
```
            logger.exception(
                ('Error: failed to execute the transaction '
                 'to prepare the %s.asset_downloads_derived PDT'), schema_name)
```

Reply:
There were indentation errors here. Also, Pythons Logging uses printf-style to avoid evaluating the string until it has to; otherwise you can get pylint errors as in:

`[logging-format-interpolation] Use % formatting in logging functions and pass the % parameters as arguments`

Reply:
fixed indenting, changed to use printf.